src/main/i18n/translations/es.ts

```diff
@@ -15,7 +15,7 @@ const translationsEs: Translations = {
 	"hide-others": "Ocultar otros",
 	"minimize": "Minimizar",
 	"paste": "Pegar",
-	"preferences": "Preferencias",
+	"preferences": "Configuración",
```

Comment:
These strings are the ones Apple uses in their apps, so I'd rather not change them.